src/model/subject.v1.yaml

```diff
@@ -19,4 +25,5 @@ allOf:
               - thumbnail
     required:
       - impactStatement
+      - aimsAndScope
```

Comment:
The property also needs a `minItems` if it's `required`. But having it required means that we'll need data being present before this can be merged. So, let's make it optional for now and can make it required in the future (which also isn't a breaking change).